pkg/controller/operators/catalog/operator.go

```diff
@@ -914,7 +903,7 @@ func (o *Operator) createInstallPlan(namespace string, subs []*v1alpha1.Subscrip
 func (o *Operator) requeueSubscription(name, namespace string) {
 	// we can build the key directly, will need to change if queue uses different key scheme
 	key := fmt.Sprintf("%s/%s", namespace, name)
-	o.subQueue.AddRateLimited(key)
+	o.subQueue.Add(key)
```

Comment:
Why was this change made?

Reply:
We shouldn't really need to rate limit our internal requeues - otherwise we can end up DOSing ourselves. In general we should only requeue when we have work to do